docs/source/llms/deployments/guides/step1-create-deployments.rst

```diff
@@ -77,29 +77,23 @@ configuration file that is defined at server start, permitting dynamic route cre
                     openai_api_key: $OPENAI_API_KEY
 
 
-Step 4: Start the Gateway
+Step 4: Start the Server
 -------------------------
-Fourth, let's test the gateway service!
+Fourth, let's test the deployments server!
 
-To launch the gateway using a YAML config file, use the gateway CLI command.
+To launch the deployments server using a YAML config file, use the deployments CLI command.
 
-The gateway will automatically start on ``localhost`` at port ``5000``, accessible via 
-the URL: ``http://localhost:5000``. To modify these default settings, use the 
-``mlflow gateway --help`` command to view additional configuration options.
+The deployments server will automatically start on ``localhost`` at port ``5000``, accessible via
+the URL: ``http://localhost:5000``. To modify these default settings, use the
+``mlflow deployments start-server --help`` command to view additional configuration options.
 
 .. code-section::
 
-    .. code-block:: bash 
-        :name: start-gateway
-
-        mlflow gateway start --config-path config.yaml 
-
+    .. code-block:: bash
+        :name: start-server
 
-.. figure:: ../../../_static/images/tutorials/gateway/creating-first-gateway/start_gateway.gif
```

Comment:
Can we delete this gif?

Reply:
It's actually painful to update it every time we update the content.

Reply:
`start_gateway.gif` is still used in the gateway docs. I think we need to keep it.